src/main/java/eu/dissco/orchestration/backend/service/SourceSystemService.java

```diff
@@ -30,4 +38,20 @@ public SourceSystemRecord updateSourceSystem(String id, SourceSystem sourceSyste
     repository.updateSourceSystem(sourceSystemRecord);
     return sourceSystemRecord;
   }
+
+  public JsonApiWrapper getSourceSystemRecords(int pageNum, int pageSize, String path){
+    var ssRecords = repository.getSourceSystems(pageNum, pageSize);
+    var dataNode = wrapData(ssRecords);
+    return new JsonApiWrapper(dataNode, new JsonApiLinks(path));
```

Comment:
Does this comply to the pagination specification in JSON:API?
https://jsonapi.org/format/#fetching-pagination
I think there should be link to next (if applicable) and previous (if applicable) 
`last` we discussed before and we ended that for the last page we would not show `next` if there is no next page

Reply:
Oh good catch, linksNode has self, first, and next and prev if applicable